packages/components/src/RecurringSelect/RecurringSelect.stories.mdx

```diff
@@ -89,4 +89,5 @@ the type of selection they are making.
 ## Related components
 
 To allow the user to select specific dates, use
-[Datepicker](/components/date-picker) or [InputDate](/components/input-date).
+[Datepicker](../?path=/docs/components-datepicker--date-picker) or
```

Comment:
`DatePicker links are strange - without the `#date-picker` part it loads halfway down the DatePicker component page?
```suggestion
[DatePicker](../?path=/docs/components-datepicker--date-picker#date-picker) or
```

Reply:
That's weird... 🤔